docs/source/how-tos/To-do-list-for-New-MOC-Intern-and-Member.md

```diff
@@ -1,62 +1,100 @@
-## To Do List for New MOC Intern and Member
+# To Do List for New MOC Intern and Member
 
-In order to become part of the MOC team, there are a few things that you need to do:
+## Things to do
+
+### For everyone
+
+1. Be able to run IRC and Slack
+  - #moc, #openinfralabs, #openinfralabs-monitoring on freenode.
+  (usually folks use ZNC bouncer, some of us use irccloud which as a built in bouncer - #openinfralabs is archived so a bouncer may not be needed.)
+
+2. Checkout docs.massopen.cloud
+  If you find things which may be improved you are encouraged to do a pull request and fix them.
+  Understand how to use SSH and generate their public keys.
+
+3. Share contact info and subscribe to the team mailing list.
  -  Add your name and contact information at [this URL](../contacts/People.html)
  -  Subscribe to the [team mailing lists](https://lists.massopen.cloud)
- -  **Trello** -  : Send mail to msd@bu.edu with your Trello user name and you will be added to the appropriate organization.
- -  Once you are a member of the MOC team on Trello, go to the 
- [moc-massopencloud](https://trello.com/b/2IMwDpdi/moc-massopencloud) Trello board, and add a Trello card on the Backlog Trello list with
-     1. your full name
-     1. short paragraph about yourself
-     1. a photo of yourself
-
-See [example](https://trello.com/c/NF4xQCuq/102-sirine-benbrahim)
-
-### Team calendars
-We use Google calendars as a collaboration tool. We have the following calendars:
- -  **MOC** -  administrative
- -  **MOC - Activities** -  team-wide activities and meetings
- -  **MOC - Engineers** -  MOC engineers and embedded engineers' office hours, 
- OpenStack meetings that engineers attend on regular basis, etc.
- -  **MOC - Interns** -   interns/students' hours working as part of the MOC.
-
-Please share your the time slots when you plan to spend time in the MOC and/or on the MOC related projects.
-Send a request to msd@bu.edu to be added to the appropriate calendar.
+
+4. **Trello**: Send mail to msd@bu.edu with your Trello user name and you will be added to the appropriate organization.
+
+5. TODO: Contact Jen on how she would like to gather information on new people to publish to the website.
```

Comment:
Should this be here in the docs? Or should it be moved to an issue or trello card?

Reply:
I emailed Jen about this but I haven't heard back yet. I will have to bug @msdisme for this one.

Reply:
I removed this section from the doc here until we have a procedure in place.